### PR TITLE
Document with tests

### PR DESCRIPTION
**Pull request by moonso**
This PR updates some documentation and docstrings

**Review:**
- [x] code approved by
- [x] tests executed by travis

### CODE REVIEW

File: docs/admin-guide/annotations.md [hunk 1]
```diff
@@ -74,7 +75,7 @@ The [PolyPhen][polyphen]) prediction for how a variation affects the protein.
 
 #### Rank score ####
 
-The combined rank score for a variant
+The combined rank score for a variant. For exact info see [test][rank_score_test]
```
Comment (dnil) [inline docs/admin-guide/annotations.md]: :+1:

File: docs/admin-guide/annotations.md [hunk 2]
```diff
@@ -137,4 +138,7 @@ What variants is this variant in Autosomal Recessive Compound with?
 [phastcons]: http://compgen.cshl.edu/phast/
 [phylop]: http://genome.ucsc.edu/cgi-bin/hgTrackUi?db=hg19&g=cons100way
 [sift]: http://sift.jcvi.org
-[polyphen]: http://genetics.bwh.harvard.edu/pph2/dokuwiki/
\ No newline at end of file
+[polyphen]: http://genetics.bwh.harvard.edu/pph2/dokuwiki/
+[polyphen]: http://genetics.bwh.harvard.edu/pph2/dokuwiki/
```
Comment (dnil) [inline docs/admin-guide/annotations.md]: Duplicate..

File: scout/parse/case.py [hunk 3]
```diff
@@ -221,7 +221,7 @@ def parse_individual(sample):
     ind_info['confirmed_sex'] = sample.get('confirmed_sex')
     ind_info['predicted_ancestry'] = sample.get('predicted_ancestry')
 
-    ind_info['bam_file'] = sample.get('bam_path')
+    ind_info['bam_file'] = sample.get('bam_path',sample.get('bam_file'))
```
Comment (dnil) [inline scout/parse/case.py]: *cringe*

Comment (dnil) [reply scout/parse/case.py]: What we really could use is listing unused/unrecognized config key-value pairs.

File: tests/conftest.py [hunk 4]
```diff
@@ -867,7 +867,7 @@ def __init__(self):
             self.ID = '.'
             self.QUAL = None
             self.var_type = 'snp'
-            self.INFO = {'RankScore': "123:10"}
+            self.INFO = {'RankScore': "internal_id:10"}
```
Comment (dnil) [inline tests/conftest.py]: :+1:

**Comment by moonso** [comment]
👯

**Pull request merged**